### PR TITLE
Update man pages for Organization commands

### PR DESCRIPTION
**Pull request by KevinAnsleyJohnson**
Signed-off-by: Kevin Johnson <kevin_johnson@cargill.com>

### CODE REVIEW

**Review by dnewh — APPROVED** [review]

**Review by vaporos — CHANGES_REQUESTED** [review]
See comments on #736, which would apply here as well.

File: cli/man/grid-organization-update.1.md [hunk 1]
```diff
@@ -43,20 +43,21 @@ OPTIONS
...
-: List of locations associated with this organization
+: List of comma-separated locations associated with this organization
 
 `--metadata`
 : Key-value pairs (format: <key>=<value>) in a comma-separated list
 
 `--service-id`
 : The ID of the service the payload should be sent to; required if running on
-Splinter. Format <circuit-id>::<service-id>
+  Splinter. Format <circuit-id>::<service-id>.
```
Comment (dplumb94) [inline cli/man/grid-organization-update.1.md:60]: Would be nice to standardize the format descriptions. We have `(format: <id_type>:<id>)` and `Format <circuit-id>::<service-id>`

Comment (KevinAnsleyJohnson) [reply cli/man/grid-organization-update.1.md:61]: done!

File: cli/man/grid-organization-create.1.md [hunk 2]
```diff
@@ -13,67 +13,71 @@ NAME
...
 =======
 
 `--alternate-ids`
-: Alternate IDs for organization
+: Alternate IDs for organization in a comma-separated list. 
+  Format: <id_type>:<id>.
 
 `-k`, `--key`
-: Base name for private signing key file
+: base name or path to a private signing key file.
```
Comment (dplumb94) [inline cli/man/grid-organization-create.1.md:50]: The b should be capitalized here

**Review by dplumb94 — CHANGES_REQUESTED** [review]

**Review by dplumb94 — APPROVED** [review]

**Review by dplumb94 — APPROVED** [review]

**Review by vaporos — APPROVED** [review]

**Pull request merged**